### PR TITLE
cdc: reformat Changefeed config and Server config

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

**This PR is a first step.** It converts the existing format of code blocks to the body heading format for config descriptions in [CLI and Configuration Parameters of TiCDC Changefeeds | TiDB Docs](https://docs.pingcap.com/tidb/dev/ticdc-changefeed-config) and [TiCDC Server Configurations | TiDB Docs](https://docs.pingcap.com/tidb/dev/ticdc-server-config), like that in [TiDB Configuration File | TiDB Docs](https://docs.pingcap.com/tidb/dev/tidb-configuration-file).

This PR doesn't add new descriptions, except describe some config value as 示例值 for now, because those config items have no description or default value. **Some missing explanations need to be added in new PRs.**

cdc: reformat Changefeed config and Server config
ref: https://github.com/pingcap/docs/issues/19522 and https://github.com/pingcap/docs/issues/19521
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs/issues/19005

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:1]: Preview: https://pingcap-docsite-preview-git-preview-pi-364f09-oreoxmts-projects.vercel.app/zh/tidb/dev/ticdc-changefeed-config

File: ticdc/ticdc-server-config.md
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:1]: Preview: https://pingcap-docsite-preview-git-preview-pi-364f09-oreoxmts-projects.vercel.app/zh/tidb/dev/ticdc-server-config

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+- 如果要关闭 bootstrap 消息的发送，则将 `send-bootstrap-interval-in-sec` 和 `send-bootstrap-in-msg-count` 均设置为 0。
+
+#### `send-bootstrap-to-all-partition`
+
+- 控制是否发送 bootstrap 消息到所有的 partition。
+- 如果设置为 `false`，则只发送 bootstrap 消息到对应表 topic 的第一个 partition。
+
+[//]: # (- 示例值 `true`)
+
+#### sink.kafka-config.codec-config`
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:318]:
```suggestion
#### sink.kafka-config.codec-config
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+### sink.pulsar-config
+
+以下配置项仅在下游为 Pulsar 时生效。
+
+#### `authentication-token`
+
+- 使用 token 进行 Pulsar 服务端的认证，此处为 token 的值。
+
+#### `token-from-file=`
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:479]:
```suggestion
#### `token-from-file`
```

File: ticdc/ticdc-server-config.md
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:1]: Preview: https://pingcap-docsite-preview-git-preview-pi-364f09-oreoxmts-projects.vercel.app/zh/tidb/dev/ticdc-server-config

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
 
 本章节详细介绍了同步任务的配置。
 
+### `memory-quota`
+
+- 指定该 Changefeed 在 Capture Server 中内存配额的上限。对于超额使用部分，会在运行中被 Go runtime 优先回收。
+- 默认值：`1073741824`，即 1 GB
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:42]:
```suggestion
- 默认值：`1073741824`，即 1 GiB
```

File: ticdc/ticdc-changefeed-config.md [hunk 4]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+### `enable-sync-point` <span class="version-mark">从 v6.3.0 版本开始引入</span>
+
+- 是否开启 Syncpoint 功能，从 v6.3.0 开始支持，该功能默认关闭。
+- 从 v6.4.0 开始，使用 Syncpoint 功能需要同步任务拥有下游集群的 SYSTEM_VARIABLES_ADMIN 或者 SUPER 权限。
+- 该参数只有当下游为 TiDB 时，才会生效。
+- 默认值：`false`
+
+### `sync-point-interval`
+
+- Syncpoint 功能对齐上下游 snapshot 的时间间隔。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:59]:
```suggestion
- Syncpoint 功能对齐上下游快照的时间间隔。
```

File: ticdc/ticdc-changefeed-config.md [hunk 5]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+- 默认值：`"24h"`
+
+### `sql-mode` <span class="version-mark">从 v6.5.6、v7.1.3 和 v7.5.0 版本开始引入</span>
+
+- 用于设置解析 DDL 时使用的 [SQL 模式](/sql-mode.md)，多个模式之间用逗号分隔。
+- 默认值：`ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`，与 TiDB 的默认 SQL 模式一致
+
+### `bdr-mode`
+
+- 如果要使用 TiCDC 搭建 BDR 集群，需要将该参数设置为 true，同时要将 TiDB 集群设置为 BDR 模式。详情请参考 [TiCDC 双向复制](/ticdc/ticdc-bidirectional-replication.md#ticdc-双向复制)
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:79]:
```suggestion
- 如果要使用 TiCDC 搭建 BDR (Bi-directional Replication) 集群，需要将该参数设置为 `true`，同时要将 TiDB 集群设置为 BDR 模式。详情请参考 [TiCDC 双向复制](/ticdc/ticdc-bidirectional-replication.md#ticdc-双向复制)
```

File: ticdc/ticdc-changefeed-config.md [hunk 6]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+#### filter.event-filters
+
+##### `matcher`
+
+- matcher 是一个白名单，`matcher = ["test.worker"]` 表示该过滤规则只应用于 `test` 库中的 `worker` 表。
+
+##### `ignore-event`
+
+- `ignore-event = ["insert"]` 表示过滤掉 insert 事件。
+- `ignore-event = ["drop table", "delete"]` 表示忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件。需要注意的是，在更新 TiDB 中聚簇索引的列值时，TiCDC 会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法将该类事件识别为 UPDATE 事件，因此无法正确地进行过滤。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:120]:
```suggestion
- `ignore-event = ["insert"]` 表示过滤掉 INSERT 事件。
- `ignore-event = ["drop table", "delete"]` 表示忽略 `DROP TABLE` 的 DDL 事件和 `DELETE` 类型的 DML 事件。需要注意的是，在更新 TiDB 中聚簇索引的列值时，TiCDC 会将一个 `UPDATE` 事件拆分成为 `DELETE` 和 `INSERT` 事件，TiCDC 无法将该类事件识别为 `UPDATE` 事件，因此无法正确地进行过滤。
```

File: ticdc/ticdc-changefeed-config.md [hunk 7]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+- `ignore-delete-value-expr = "name = 'john'"` 表示过滤掉包含 `name = 'john'` 条件的 delete DML。
+
+##### `ignore-insert-value-expr`
+
+- `ignore-insert-value-expr = "id >= 100"` 表示过滤掉包含 id >= 100 条件的 insert DML。
+
+##### `ignore-update-old-value-expr`
+
+- `ignore-update-old-value-expr = "age < 18"` 表示过滤掉旧值 `age < 18` 的 update DML。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:136]:
```suggestion
- `ignore-sql = ["^drop", "add column"]` 表示过滤掉以 `DROP` 开头或者包含 `ADD COLUMN` 的 DDL。

##### `ignore-delete-value-expr`

- `ignore-delete-value-expr = "name = 'john'"` 表示过滤掉包含 `name = 'john'` 条件的 `DELETE` DML。

##### `ignore-insert-value-expr`

- `ignore-insert-value-expr = "id >= 100"` 表示过滤掉包含 id >= 100 条件的 `INSERT` DML。

##### `ignore-update-old-value-expr`

- `ignore-update-old-value-expr = "age < 18"` 表示过滤掉旧值 `age < 18` 的 `UPDATE` DML。
```

File: ticdc/ticdc-changefeed-config.md [hunk 8]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+##### `ignore-update-new-value-expr`
+
+- `ignore-update-new-value-expr = "gender = 'male'"` 表示过滤掉新值 `gender = 'male'` 的 update DML。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:140]:
```suggestion
- `ignore-update-new-value-expr = "gender = 'male'"` 表示过滤掉新值 `gender = 'male'` 的 `UPDATE` DML。
```

File: ticdc/ticdc-changefeed-config.md [hunk 9]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+### scheduler
+
+#### `enable-table-across-nodes`
+
+- 将表以 Region 为单位分配给多个 TiCDC 节点进行同步。
+- 该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
+- `enable-table-across-nodes` 开启后，有两种分配模式：
+
+    1. 按 Region 的数量分配，即每个 CDC 节点处理 region 的个数基本相等。当某个表 Region 个数大于 `region-threshold` 值时，会将表分配到多个节点处理。`region-threshold` 默认值为 10000。
+    2. 按写入的流量分配，即每个 CDC 节点处理 region 总修改行数基本相当。只有当表中每分钟修改行数超过 `write-key-threshold` 值时，该表才会生效。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:151]:
```suggestion
    1. 按 Region 的数量分配，即每个 TiCDC 节点处理 Region 的个数基本相等。当某个表 Region 个数大于 `region-threshold` 值时，会将表分配到多个节点处理。`region-threshold` 默认值为 `10000`。
    2. 按写入的流量分配，即每个 TiCDC 节点处理 Region 总修改行数基本相当。只有当表中每分钟修改行数超过 `write-key-threshold` 值时，该表才会生效。
```

File: ticdc/ticdc-changefeed-config.md [hunk 10]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+[//]: # (以下是 MQ 类型 sink 配置)
+
+#### `dispatchers`
+
+- 对于 MQ 类的 Sink，可以通过 dispatchers 配置 event 分发器。
+- 支持 partition 及 topic（从 v6.1 开始支持）两种 event 分发器。二者的详细说明见下一节。
+- matcher 的匹配语法和过滤器规则语法相同，matcher 匹配规则的详细说明见下一节。
+- 该参数只有当下游为消息队列时，才会生效。
+- 当下游 MQ 为 Pulsar 时，如果 partition 的路由规则未指定为 `ts`、`index-value`、`table`、`default` 中的任意一个，那么将会使用你设置的字符串作为每一条 Pulsar message 的 key 进行路由。例如，如果你指定的路由规则为 'code' 字符串，那么符合该 matcher 的所有 Pulsar message 都将会以 'code' 作为 key 进行路由。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:176]:
```suggestion
- 当下游 MQ 为 Pulsar 时，如果 partition 的路由规则未指定为 `ts`、`index-value`、`table`、`default` 中的任意一个，那么将会使用你设置的字符串作为每一条 Pulsar message 的 key 进行路由。例如，如果你指定的路由规则为 `'code'` 字符串，那么符合该 matcher 的所有 Pulsar message 都将会以 `'code'` 作为 key 进行路由。
```

File: ticdc/ticdc-changefeed-config.md [hunk 11]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+- 控制是否发送 bootstrap 消息到所有的 partition。
+- 如果设置为 `false`，则只发送 bootstrap 消息到对应表 topic 的第一个 partition。
+
+[//]: # (- 示例值 `true`)
+
+#### sink.kafka-config.codec-config
+
+##### `encoding-format`
+
+- 用来控制 simple protocol 的消息的编码格式，目前支持 "json" 和 "avro" 两种格式。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:322]:
```suggestion
- 用来控制 simple protocol 的消息的编码格式，目前支持 `json` 和 `avro` 两种格式。
```

File: ticdc/ticdc-changefeed-config.md [hunk 12]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+- 数据一致性级别。设置为 `"none"` 时将关闭 redo log。
+- 默认值：`"none"`
+- 可选值：`"none"`、`"eventual"`
+
+#### `max-log-size`
+
+- redo log 的最大日志大小。
+- 默认值：`64`
+- 单位：MB
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:356]:
```suggestion
- 单位：MiB
```

File: ticdc/ticdc-changefeed-config.md [hunk 13]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+- 默认值：`""`
+
+#### `sasl-oauth-scopes`
+
+- Kafka SASL OAUTHBEARER 认证机制中的 scopes。在使用该认证机制时，该参数可选填。
+- 默认值：`""`
+
+#### `sasl-oauth-grant-type`
+
+- Kafka SASL OAUTHBEARER 认证机制中的 grant-type。默认值为 "client_credentials"。在使用该认证机制时，该参数可选填。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:442]:
```suggestion
- Kafka SASL OAUTHBEARER 认证机制中的 grant-type。默认值为 `"client_credentials"`。在使用该认证机制时，该参数可选填。
```

File: ticdc/ticdc-server-config.md [hunk 14]
```diff
@@ -28,61 +28,149 @@ summary: 了解 TiCDC 详细的命令行参数和配置文件定义。
...
+- 单位：MiB
+
+##### `max-days`
+
+- log 文件最长保留天数。可选。
+- 默认值：`0`，代表永不删除
+
+##### `max-backups`
+
+- log 文件的保留个数。可选。
```
Comment (hfxsd) [inline ticdc/ticdc-server-config.md:144]:
```suggestion
- 日志文件的保留个数。可选。
```

File: ticdc/ticdc-server-config.md [hunk 15]
```diff
@@ -28,61 +28,149 @@ summary: 了解 TiCDC 详细的命令行参数和配置文件定义。
...
+- 默认值：`0`，代表保留所有 log 文件
```
Comment (hfxsd) [inline ticdc/ticdc-server-config.md:145]:
```suggestion
- 默认值：`0`，代表保留所有日志文件
```

File: ticdc/ticdc-changefeed-config.md [hunk 16]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+### `bdr-mode`
+
+- 如果要使用 TiCDC 搭建 BDR 集群，需要将该参数设置为 true，同时要将 TiDB 集群设置为 BDR 模式。详情请参考 [TiCDC 双向复制](/ticdc/ticdc-bidirectional-replication.md#ticdc-双向复制)
+- 默认值：`false`，表示不处于 BDR 模式
+
+### `changefeed-error-stuck-duration`
+
+- changefeed 发生内部错误或异常时允许自动重试的时间。
+- 若 changefeed 发生内部错误或异常，且持续时间超过该参数设置的时间，changefeed 会进入 Failed 状态。
+- 当 changefeed 处于 failed 状态时，需要手动重启 changefeed 才能恢复。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:86]:
```suggestion
- Changefeed 发生内部错误或异常时允许自动重试的时间。
- 若 Changefeed 发生内部错误或异常，且持续时间超过该参数设置的时间，Changefeed 会进入 Failed 状态。
- 当 Changefeed 处于 Failed 状态时，需要手动重启 Changefeed 才能恢复。
```

File: ticdc/ticdc-changefeed-config.md [hunk 17]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+##### `output-handle-key`
+
+- 是否输出 handle 列信息。该配置项仅用于内部实现，不推荐设置该配置项。
+- 默认值：`false`
+
+##### `output-old-value`
+
+- 是否输出行数据更改前的值。
+- 开启后，Update 事件会输出两行数据：第一行为 Delete 事件，输出更改前的数据；第二行为 Insert 事件，输出更改后的数据。
+- 开启后，即当该参数设为 true 时，会在变更数据列前增加 "is-update" 列。该列用来标识当前行的变更数据是来自 Update 事件，还是原始的 Insert/Delete 事件。如果当前行的变更数据来自 Update 事件，则 "is-update" 列为 true，否则为 false。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:292]:
```suggestion
- 开启后，UPDATE 事件会输出两行数据：第一行为 DELETE 事件，输出更改前的数据；第二行为 INSERT 事件，输出更改后的数据。
- 开启后，即当该参数设为 `true` 时，会在变更数据列前增加 `"is-update"` 列。该列用来标识当前行的变更数据是来自 Update 事件，还是原始的 INSERT /DELETE 事件。如果当前行的变更数据来自 UPDATE 事件，则 `"is-update"` 列为 `true`，否则为 `false`。
```

File: ticdc/ticdc-changefeed-config.md [hunk 18]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+- 控制发送 bootstrap 消息的时间间隔。
+- 默认值：`120`，即每张表每隔 120 秒发送一次 bootstrap 消息
+- 单位：秒
+
+#### `send-bootstrap-in-msg-count`
+
+- 控制发送 bootstrap 的消息间隔，单位为消息数。
+- 默认值：`10000`，即每张表每发送 10000 条行变更消息就发送一次 bootstrap 消息
+- 如果要关闭 bootstrap 消息的发送，则将 `send-bootstrap-interval-in-sec` 和 `send-bootstrap-in-msg-count` 均设置为 0。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:309]:
```suggestion
- 如果要关闭 bootstrap 消息的发送，则将 `send-bootstrap-interval-in-sec` 和 `send-bootstrap-in-msg-count` 均设置为 `0`。
```

File: ticdc/ticdc-changefeed-config.md [hunk 19]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+
+- 用来控制 simple protocol 的消息的编码格式，目前支持 "json" 和 "avro" 两种格式。
+- 默认值：`json`
+- 可选值：`json`、`avro`
+
+#### sink.open
+
+##### `output-old-value`
+
+- 是否输出行数据更改前的值。关闭后，Update 事件不会输出 "p" 字段的数据。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:330]:
```suggestion
- 是否输出行数据更改前的值。关闭后，UPDATE 事件不会输出 "p" 字段的数据。
```

File: ticdc/ticdc-changefeed-config.md [hunk 20]
```diff
@@ -36,332 +36,570 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
...
+- 默认值：`true`
+
+#### sink.debezium
+
+##### `output-old-value`
+
+- 是否输出行数据更改前的值。关闭后，Update 事件不会输出 "before" 字段的数据。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:337]:
```suggestion
- 是否输出行数据更改前的值。关闭后，UPDATE 事件不会输出 "before" 字段的数据。
```

File: ticdc/ticdc-server-config.md [hunk 21]
```diff
@@ -28,61 +28,149 @@ summary: 了解 TiCDC 详细的命令行参数和配置文件定义。
...
+
+#### `cache-size-in-mb`
+
+- Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小。
+- 默认值：`128`
+- 单位：MiB
+
+#### `sorter-dir`
+
+- Sorter 文件相对于 data-dir 的目录。可选。
```
Comment (hfxsd) [inline ticdc/ticdc-server-config.md:157]:
```suggestion
- Sorter 文件相对于 `data-dir` 的目录。可选。
```

File: ticdc/ticdc-server-config.md [hunk 22]
```diff
@@ -28,61 +28,149 @@ summary: 了解 TiCDC 详细的命令行参数和配置文件定义。
...
+#### `error-output`
+
+- 用于指定 zap log 模块内部的错误日志的输出位置。可选。
+- 默认值：`"stderr"`
+
+#### log.file
+
+##### `max-size`
+
+- 单个 log 文件的最大文件大小。可选。
```
Comment (hfxsd) [inline ticdc/ticdc-server-config.md:133]:
```suggestion
- 单个日志文件的最大文件大小。可选。
```

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md (hunk 12 above)
Comment (Oreoxmt) [reply ticdc/ticdc-changefeed-config.md:356]: https://github.com/pingcap/tiflow/blob/19fc702ce546f556586b8466063911bd537250e3/pkg/config/consistent.go#L33-L35

```go
// MaxLogSize is the max size(MiB) of a log file written by redo log.
// Default is 64MiB.
MaxLogSize int64 `toml:"max-log-size" json:"max-log-size"`
```

**Review by flowbehappy — APPROVED** [review]

**Pull request merged**